sale_order_partner_restrict/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 {
     "name": "Sale Order Partner Restrict",
     "summary": "Apply restrictions when selecting from the list of customers on SO.",
-    "version": "14.0.1.0.1",
+    "version": "14.0.1.0.2",
```

Comment:
@dessanhemrayev no need to update manually

Reply:
Okay, I'll fix it